Makefile

```diff
@@ -156,7 +156,7 @@ BINDATA_DIRS := \
 
 .PHONY: compile
 compile:
-	$(GOVENDOR) install \
+	CGO_CFLAGS_ALLOW='.*' CGO_LDFLAGS_ALLOW='.*' $(GOVENDOR) install \
```

Comment:
Why not add CGO_... settings as part of definision of GOVENDOR variable:

GOVENDOR := CGO_CFLAGS_ALLOW='.*'  ... govendor